oral_history/templates/oral_history/error.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+
+There was an error upload the form. Please try again and make sure you fill out every form and entry lengths aren't too long.
```

Comment:
should be more clear

Reply:
forms now refresh to show individual entry errors. done.